tests/conftest.py

```diff
@@ -246,7 +247,6 @@ def unicode_emojis():
     'reactions': [],
     'type': 'private',
     'avatar_url': 'dummy_avatar_url',
-    'subject_links': [],
```

Comment:
Ideally, this shouldn't be the case :thinking:, as there's no such 'topic' in PMs (or group PMs) as you mentioned earlier. This could be potentially filed in the main repo, it not before, in any case.